src/Components/CreateImageWizardV2/CreateImageWizard.scss

```diff
@@ -82,3 +82,7 @@ div.pf-v5-c-alert.pf-m-inline.pf-m-plain.pf-m-warning {
         margin-block-start: 0;
     }
 }
+
+.pf-v5-c-pagination.pf-m-bottom .pf-v5-c-menu-toggle {
+    display: flex
+}
```

Comment:
Just a heads up - the position of the arrow in bottom pagination is a bug directly in PF, they know about it and are working on releasing a quick patch fix for it.

Which means we'll be probably able to remove it :)

Reply:
Will add a note to check to remove it.

Reply:
Perfect, thanks!